packages/fiori/src/ShellBar.js

```diff
@@ -1120,7 +1098,7 @@ class ShellBar extends UI5Element {
 	}
 
 	get menuPopoverExpanded() {
-		return this.hasMenuItems ? this._menuPopoverExpanded : undefined;
+		return HasPopup.Menu;
```

Comment:
We should keep the expresion, because if the menu popover the button will persist without popover. With this logic it will have always haspop attribute even if it is not needed. Also i think this should be moved to menuBtnHasPopup function

Reply:
You are right. After our discussion I've checked the specification and retrieved all internal properties related to the aria-expanded attributes of both ui5-shellbar menus. The only difference now would be that aria-haspopup attribute value will be "menu" instead of "true", as "menu" is one of the documented values for the new <code>accessibilityAttributes</code> property. According to the ACC spec, both "menu" or "true"/"false" are valid values for the aria-haspopup attribute. Not currently sure which is more suitable.